LoRaEngine/modules/LoRaWanNetworkSrvModule/Logger/Logger.cs

```diff
@@ -66,7 +66,7 @@ public static void Log(string message, LogLevel logLevel)
 
         public static void Log(string deviceId, string message, LogLevel logLevel)
         {
-            if ((int)logLevel >= configuration.LogLevel)
+            if ((int)logLevel >= (int)configuration.LogLevel)
```

Comment:
My bad, for some reason I thought the Enum values were not in correct order.
nit: you don't need to cast to int before comparing

Reply:
Updated!